src/shared-react/hooks.ts

```diff
@@ -6,7 +6,18 @@ import { i18nConfig } from '../shared/constants';
 export type TFunc = TFunction<typeof i18nConfig.ns, string>;
 
 export function useT(prefix?: string): TFunc {
-  return useTranslation(i18nConfig.ns, { keyPrefix: prefix }).t;
+  const { t, i18n } = useTranslation(i18nConfig.ns, { keyPrefix: prefix });
+  return (...args: Parameters<typeof t>) => {
```

Comment:
I know what this is for only after reading the commit title. How about a one-liner comment here?